src/server/rest/v1/service/TransactionService.ts

```diff
@@ -1071,4 +1085,20 @@ export default class TransactionService {
     );
     return transactions;
   }
+
+  private static async getTransactionsCompleted(action: ServerAction, req: Request): Promise<DataResult<Transaction>> {
+    return await TransactionService.getTransactions(req, action, { completedTransactions: true }, [
+      'id', 'chargeBoxID', 'timestamp', 'issuer', 'stateOfCharge', 'timezone', 'connectorId', 'meterStart', 'siteAreaID', 'siteID',
+      'stop.roundedPrice', 'stop.price', 'stop.priceUnit', 'stop.inactivityStatus', 'stop.stateOfCharge', 'stop.timestamp', 'stop.totalConsumptionWh',
+      'stop.totalDurationSecs', 'stop.totalInactivitySecs', 'stop.meterStop', 'billingData.invoiceID', 'ocpi', 'ocpiWithCdr', 'tagID', 'stop.tagID',
+    ]);
+  }
+
+  private static async getTransactionsActive(action: ServerAction, req: Request): Promise<DataResult<Transaction>> {
+    return await TransactionService.getTransactions(req, action, { completedTransactions: false }, [
```

Comment:
Set the query string Status param to 'active' and call the getTransactions() in which you can provide a IF to add the proper filter (if Status is not provided return all transactions)

Reply:
I understand how to retrieve completed or active transactions from the same getTransactions() method. However I can't figure out how to return both from the same call, as they need different projections, without breaking pagination.

Reply:
Done